### ISSUE TITLE
Missing preview for codes-of-conduct

### ISSUE THREAD

**Issue by WillAbides** [octokit/routes#299]
- endpoint scope: codesOfConduct
- endpoint name: List all codes of conduct
- route: `GET /codes_of_conduct`
- documentation URL: https://developer.github.com/v3/codes_of_conduct/#list-all-codes-of-conduct

### Current specification

https://github.com/octokit/routes/blob/62aad3e4186048c1f6d7881572ff44ac2788c14e/routes/codes-of-conduct/list-all-codes-of-conduct.json#L1-L31

### I expected

The codes-of-conduct endpoints should all have the required param `scarlet-witch`.

/cc https://github.com/go-github-cli/go-github-cli/issues/27

**Comment by gr2m** [comment]
It’s so cool that you use octokit/routes! 😄 

First step to get it fixed is to edit the `routes/codes-of-conduct/list-all-codes-of-conduct.json` file to what you expect. Then run the test for only this one file

```
https://developer.github.com/v3/codes_of_conduct/#list-all-codes-of-conduct ./node_modules/.bin/tap test/integration/single-endpoint-test.js
```

That should fail now. You can then already start a pull request with the failing tests. Then we can figure out how to fix it :) 

The problem is that note about the preview header is not within the endpoint sections. We could implement a workaround and at the same time ask the docs team to update the docs page, at which point we can remove the workaround?

**Comment by WillAbides** [comment]
> It’s so cool that you use octokit/routes!

Thanks for working on this.  It's been a huge help.  I started out thinking I would use it just to fill in the cli help, but then I realized I could generate all the endpoints from routes.json.

> The problem is that note about the preview header is not within the endpoint sections

I see...I was focused on why the regex wouldn't match but I didn't think to check where the div is. 

I'll see about moving the preview div into endpoints.  If that takes too long, I'll look at the workaround.
